### PR TITLE
Add MSP support for led profile and status

### PR DESCRIPTION
**Pull request by haslinghuis**
Finally add support for https://github.com/betaflight/betaflight/pull/7491

Added a flag indicating the level of ledstrip support (to allow improved Configurator behavior) along with support for reading/setting the current ledstrip_profile `[RACE, BEACON, STATUS]`

ledstrip support flag `[STATUS]` is as follows:
0 = basic ledstrip available
1 = advanced ledstrip available

In follow up we could do something useful with this.

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]
* approving untested

**Comment by ctzsnooze** [comment]
The distinction between basic and complex profiles is no longer relevant.
The whole LED code needs to be refactored without that distinction.
We should not change MSP to add a new element now, because it is likely to be unnecessary when the LED code is properly modernised.

**Comment by haslinghuis** [comment]
This is not a change. Just implementing current firmware behavior. Any changes to that can be done in subsequent PR.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
